splunk_eventgen/eventgen_core.py

```diff
@@ -300,8 +288,8 @@ def _proc_worker_do_work(work_queue, logging_queue, config):
         root = logging.getLogger()
         root.setLevel(logging.DEBUG)
         if logging_queue is not None:
-            # TODO https://github.com/splunk/eventgen/issues/217
-            qh = logutils.queue.QueueHandler(logging_queue)
```

Comment:
Remove `logutils` from source code and replace `QueueHandler` with `logging.handlers.QueueHandler`.